code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java

```diff
@@ -0,0 +1,97 @@
+package io.cattle.platform.api.auth;
+
+import io.github.ibuildthecloud.gdapi.annotation.Field;
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(name = "identity", pluralName = "identities")
+public class Identity {
+
+    private final String externalId;
+    private final String profilePicture;
+    private final String name;
+    private final String kind;
+    private final String profileUrl;
```

Comment:
If profilePicture and profileUrl are URLs then they should probably be returned in the links section of the object